### PR TITLE
Update ISSUE_TEMPLATE to match new handbook

### PR DESCRIPTION
**Pull request by seanh**
Update the issue template to match the new **Getting work ready for the Product Team** document in the new handbook repo: <https://github.com/hypothesis/handbook/blob/master/product-team/getting-work-ready-for-the-team.md>

### CODE REVIEW

File: ISSUE_TEMPLATE [hunk 1]
```diff
@@ -1,26 +1,50 @@
-## Bug report form
-### Steps to reproduce
-1.
-2.
-3.
+<!--
+Fill out either the Bug Report or the Feature Request form below. For more about how to describe work see:
+https://github.com/hypothesis/handbook/blob/master/product-team/getting-work-ready-for-the-team.md
```
Comment (dhwthompson) [inline ISSUE_TEMPLATE:3]: Not everyone who has access to the product backlog (i.e. everyone) currently has access to the handbook. I can see this getting confusing for people outside the team. Any thoughts on how we could make it less confusing?

File: ISSUE_TEMPLATE [hunk 2]
```diff
@@ -1,26 +1,50 @@
-## Bug report form
-### Steps to reproduce
-1.
-2.
-3.
+<!--
```
Comment (dhwthompson) [inline ISSUE_TEMPLATE:1]: I like the use of HTML comments here. 👍

**Review by dhwthompson — COMMENTED** [review]
My main question about this: who’s the intended audience? Do we want to optimise for external contributors, or for team members? That might affect how much we rely on linking to a repo that’s currently private to the team, and how much we expect up-front in terms of, say, acceptance criteria.

**Comment by seanh** [comment]
Bah, github issue templates suck. We'd want different templates for different audiences, and different templates for bug vs feature, and there isn't a good way to do either. We should probably use different repos for the different purposes. For now I think this is a distraction so just going to close it.

**Pull request closed without merging**